docs/development.md

```
@@ -155,6 +155,66 @@ Individual tools:
+
+| Class | Used for |
+|-------|---------|
+| `.swh-badge-open` | Open tickets |
+| `.swh-badge-in-progress` | In Progress tickets |
+| `.swh-badge-resolved` | Resolved tickets |
+| `.swh-badge-closed` | Closed tickets |
+| `.swh-badge-sla-warn` | SLA warning state |
+| `.swh-badge-sla-breach` | SLA breach state |
+
+**PHP pattern** (in admin list / editor):
+```php
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)